mmv1/third_party/terraform/services/contactcenterinsights/resource_contact_center_insights_view_test.go

```diff
@@ -0,0 +1,108 @@
+package contactcenterinsights_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestInsightsView(t *testing.T) {
```

Comment:
Let's rename this to 
```suggestion
func TestAccContactCenterInsightsView_update(t *testing.T) {
```
for consistency with the generated examples, and to actually have it running in CI. I think it needs to be prefixed with `TestAcc`